sos/plugins/openstack_containerlogs.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2019 Red Hat, Inc., Cédric Jeanneret <cjeanner@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+import os
+
+
+class OpenStackCinder(Plugin):
+    """OpenStack container logs
+    """
+    plugin_name = "openstack_containerlog"
+    profiles = ('openstack', 'openstack_controller', 'openstack_compute')
+
+
+    def setup(self):
+        if self.get_option("all_logs"):
+            self.add_copy_spec([
+                "/var/log/containers/",
+            ])
+        else:
+            self.add_copy_spec(self._subdirs('/var/log/containers'))
+
+    def _subdirs(self, root='/var/log/containers'):
+        '''Get subdirs of passed root path
+        '''
+        dir_list = []
+        for dirName, subdirs, fname in os.walk(root):
+            if len(fname) != 0:
+                dir_list.append(os.path.join(dirName, '*.log'))
```

Comment:
Is there really nothing else there of worth that doesn't end in `.log`?

What else is being captured by the `--all-logs` option?

Reply:
hmm good question. There might be some "useless" files in some directories. In fact I took that "all_log" thingy because it was present and used in some of the openstack related plugins.